app/templates/common/base_main.html

```diff
@@ -115,6 +115,143 @@ <h6>The user agrees not to:</h6>
+            <h6>Run the tool locally</h6>
+            <p>Tools and pipelines can be run locally on any system using Boutiques and a container engine 
+            (Docker or Singularity) based on the Zenodo ID of the tool. The container image for the tool is 
+            isted in the detailed page of the tool/pipeline.</p>
+            <p>At the bottom of the pipeline page, detailed instructions on how to run the tool locally 
+            are available to the user.</p>
+            <img src="/static/img/Boutiques_instructions.png"alt="Portal_access_from_conp.ca" width="700px">
+            <p>If you have never used Boutiques in the past, we recommend the following tutorial.</p>
+            <h6>Run the tool on HPCs</h6>
+            <p>Some tools and pipelines can also be run on HPCs via the CBRAIN infrastructure or in 
+            your Compute Canada account with Clowdr.</p>
+            <img src="/static/img/CBRAIN_link_highlighted.png"alt="Portal_access_from_conp.ca" width="700px">
+            <p>The 'Click here to begin' link under the 'Run on the CBRAIN Plateform' container will open a 
+            tab to the login page of CBRAIN or to your CBRAIN account if you are already logged in.</p>
+            <p>The 'Click here for instructions' link under the 'Run on HPCs with Clowdr' will open a 
+            new tab with the instructions to follow to deploy a tool/pipeline on your Compute Canada account.</p>
+          </ul>
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-primary" id="btnAcceptTut3">OK</button>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="modal fade" id="tutModal4" tabindex="-1" role="dialog" data-backdrop="static" data-keyboard="false"
+  aria-labelledby="modalCenterTitle" aria-hidden="true">
+  <div class="modal-dialog modal-dialog-centered modal-lg" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h4 class="modal-title w-100 text-center" id="modalLongTitle">Help</h4>
+      </div>
+      <div class="modal-body">
+        <h4>How can I upload my dataset?</h4>
+        <p>A dataset can be uploaded to CONP via the following mechanisms:</p>
+        <li>Through Zenodo by tagging your dataset with the keyword canadian-open-neuroscience-platform</li>
+        <li>Through the Open Science Framework (OSF) by tagging your dataset with the keyword canadian-open-neuroscience-platform
+         and setting the dataset to Public</li>
+         <li>Through DataLad within the context of CONP</li>
+         <p>Detailed technical instructions are available for all these options on the Share page of the portal.</p>
+         <h4>How can I upload my tool/pipeline?</h4>
+         <p>You can publish your tools/pipelines using Boutiques’ command bosh publish.</p>
+         <p>For more information on how to do that, please visit the ‘Publishing your own tool’ section of the 
+         Boutiques tutorial Python notebook.</p>
```

Comment:
```suggestion
        <h4>Share Page</h4>
        <h5>How to Share Datasets and Pipelines Tab</h5>
        <p>This tab contains detailed technical instructions on how to upload your dataset or pipeline to CONP.</p> 
        <h6>Tool/Pipeline Upload</h6>
        <p>You can publish your tools/pipelines using Boutiques’ command bosh publish.</p>
        <p>For more information on how to do that, please visit the ‘Publishing your own tool’ section of the 
         Boutiques tutorial Python notebook.</p>
         <h6>Dataset Upload</h6>
        <p>A dataset can be uploaded to CONP via the following mechanisms:</p>
        <li>Through Zenodo by tagging your dataset with the keyword canadian-open-neuroscience-platform</li>
        <li>Through the Open Science Framework (OSF) by tagging your dataset with the keyword canadian-open-neuroscience-platform
         and setting the dataset to Public</li>
         <li>Through DataLad within the context of CONP</li>
         <h5>Create or Edit a DATS.json File Tab</h5>
         <p>Each dataset to be uploaded should contain a valid DATS.json descriptors derived from schema.org.
         We provide a DATS GUI editor that will create a DATS.json file to ease the creation process of the 
         DATS.json that will describe the dataset to be uploaded</p>
```

At the bottom of this help text, could you create a screenshot of the DATS GUI editor and show it after the short description of the editor?